distributions/Dist_az1332.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from base_distribution import BaseDistribution
+
+class Dist_az1332(BaseDistribution):
+	def __init__(self):
+		self.f_max = 1
+		self.x_min = -10
+		self.x_max = 10
+
+
+	def pdf(self, x):
+		"""This is your PDF""" x
+		return x**2
+
+	def mean(self):
+		"""This is the mean of the PDF"""
+		return 0.
+
+	def std(self):
+		"""This is the standard deviation of the pdf"""
+		return 5.972581
```

Comment:
where does this number come from? You should have an equation based on calculus and x_min and x_max.